src/main/java/com/marklogic/kafka/connect/source/AbstractSourceTask.java

```diff
@@ -0,0 +1,66 @@
+package com.marklogic.kafka.connect.source;
+
+import org.apache.kafka.connect.source.SourceRecord;
+import org.apache.kafka.connect.source.SourceTask;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Base class for concrete SourceTask implementations, providing some generic functionality.
+ */
+abstract class AbstractSourceTask extends SourceTask {
+
+    protected final Logger logger = LoggerFactory.getLogger(getClass());
+
+    private boolean logKeys = false;
```

Comment:
Let's not include these yet - I'm not sure if they'll have any value anyway since we're not going to populate either on the SourceRecords that we make.